applications/GeoMechanicsApplication/tests/test_pressure_point_flux.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsTransientPressurePointFluxTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the regression on a previously obtained value.
```

Comment:
In the README file, analytical results are mentioned. The steady state value probably is an analytical value.